website/docs/docs/quickstarts/dbt-cloud/redshift-qs.md

```diff
@@ -0,0 +1,285 @@
+- You have a  [dbt Cloud account](https://cloud.getdbt.com/). 
+- You have an AWS account with permissions to execute a CloudFormation template to create appropriate roles and a Redshift cluster.
+
+## Create a Redshift cluster
+1. Sign in to your [AWS account](https://signin.aws.amazon.com/console) as a root user or an IAM user depending on your level of access.
+2. Use a CloudFormation template to quickly set up a Redshift cluster. A CloudFormation template is a configuration file that automatically spins up the necessary resources in AWS. [Start a CloudFormation stack](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=dbt-workshop&templateURL=https://tpch-sample-data.s3.amazonaws.com/create-dbtworkshop-infr) and you can refer to the [create-dbtworkshop-infr JSON file](https://github.com/aws-samples/aws-modernization-with-dbtlabs/blob/main/resources/cloudformation/create-dbtworkshop-infr) for more template details.
+
+
+3. Click **Next** for each page until you reach the **Select acknowledgement** checkbox. Select **I acknowledge that AWS CloudFormation might create IAM resources with custom names** and click **Create Stack**.  You should land on the stack page with a CREATE_IN_PROGRESS status.
+
+    <Lightbox src="/img/redshift_tutorial/images/cloud_formation_in_progress.png" title="Cloud Formation in Progress" />
+
+4. When the stack status changes to CREATE_COMPLETE, click the **Outputs** tab on the top to view information that you will use throughout the rest of this guide. Save those credentials for later by keeping this open in a tab.
+
+5. Type `Redshift` in the search bar at the top and click **Amazon Redshift**.
+
+    <Lightbox src="/img/redshift_tutorial/images/go_to_redshift.png" title="Click on Redshift" />
+
+6. Confirm that your new Redshift cluster is listed in **Cluster overview**. Select your new cluster.
+
+<Lightbox src="/img/redshift_tutorial/images/cluster_overview.png" title="Available Redshift Cluster" />
+
+7. Click **Query Data**. You can choose the classic query editor or v2. We will be using the v2 version for the purpose of this guide.
+
+8. You might be asked to Configure account. For the purpose of this sandbox environment, we recommend selecting “Configure account”.
+
+9. Select your cluster from the list and fill out the credentials from the output of the stack:
+    - **Database** &mdash; `dbtworkshop`
+    - **User name** &mdash; `dbtadmin`
+    - **Password** &mdash; Choose a password and save it for later
+
+<Lightbox src="/img/redshift_tutorial/images/redshift_query_editor.png" title="Redshift Query Editor v2" />
+
+<Lightbox src="/img/redshift_tutorial/images/connect_to_redshift_cluster.png" title="Connect to Redshift Cluster" />
+
+10. Click **Create connection**.
+
+## Load data 
+
+Now we are going to load our sample data into the S3 bucket that our Cloudformation template created. S3 buckets are a cheap and simple way to store data outside of Redshift.
```

Comment:
Sorry, I know the editorial review is later but the word `cheap` really bothers me. 
```suggestion
Now we are going to load our sample data into the S3 bucket that our Cloudformation template created. S3 buckets are a simple and inexpensive way to store data outside of Redshift.
```